content/resources/checklist-of-requirements-for-federal-digital-services.md

```diff
@@ -35,7 +35,13 @@ These high-level policies cover basic requirements for all websites and digital
 - See a full list of [policies and regulations »]({{< ref "/topics/policy" >}})
 - See our list of [**Required Web Content and Links »**](https://digital.gov/resources/required-web-content-and-links/?=checklist)
 
-<p class="more"><a href="{{< ref "/topics/21st-century-idea" >}}">More on 21st Century IDEA <i class="fas fa-arrow-alt-circle-right"></i></a></p>
+<p class="more">
+  <a href="{{< ref "/topics/21st-century-idea" >}}">More on 21st Century IDEA
+    <svg class="dg-icon--standard margin-bottom-05" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
```

Comment:
We should follow the recommended practice of also adding the `usa-icon` class to the SVG element.

Adding it to all usa-icon SVG's would allow us to easily identify USA icons apart from DG specific icons.

Reply:
This is now refactored to use `dg-icon` utility classes